### PR TITLE
Update whats new 8.7

### PR DESCRIPTION
**Pull request by felixpalmer**
For https://github.com/visgl/deck.gl/issues/6652

Image below needs to be uploaded to deck.gl-data if acceptable

masking-flights-cities-texas

### CODE REVIEW

File: docs/upgrade-guide.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 
 - `CartoBQTilerLayer` is removed. Use `CartoLayer` instead with `type` set to `MAP_TYPES.TILESET`.
 - `CartoSQLLayer` is removed. Use `CartoLayer` instead with `type` set to `MAP_TYPES.QUERY`.
+- `API_VERSIONS.V3` is the default for the CARTO module API calls. Use `setDefaultCredentials` to explictly specify `V2`.
```
Comment (borja-munoz) [inline docs/upgrade-guide.md:9]: Small typo in "explictly"

File: docs/whats-new.md [hunk 2]
```diff
@@ -2,6 +2,33 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
+## deck.gl v8.7
+
+Release date: February 24, 2022
+
+### MaskExtension
+
+The new [MaskExtension](/docs/api-reference/extensions/mask-extension) allows layers to show/hide objects by a geofence. For example, a map may filter a list of user locations by the boundaries of a given country, or highlight part of a base map that is inside a user-drawn circle or lasso area.
```
Comment (borja-munoz) [inline docs/whats-new.md:11]: The documentation page for MaskExtension exists but it is not linked in the table of contents.

File: docs/whats-new.md [hunk 3]
```diff
@@ -2,6 +2,33 @@
...
+
+![Masking flights and cities in Texas](https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/googlemaps-scenegraph.png?raw=true)
+
+### @deck.gl/carto fetchMap
+
+[fetchMap](/docs/api-reference/carto/fetch-map) is added to the [CARTO deck.gl submodule](github.io/deck.gl/docs/api-reference/carto/overview), greatly streamlining the display of maps created using CARTO 3.
```
Comment (borja-munoz) [inline docs/whats-new.md:17]: I think the github.io link should point directly to /docs/api-reference...

**Review by borja-munoz — CHANGES_REQUESTED** [review]
Looks good to me, just minor change requests

**Review by ibgreen — CHANGES_REQUESTED** [review]
Please put company-specific integration updates at the bottom.

File: docs/upgrade-guide.md [hunk 4]
```diff
@@ -6,6 +6,7 @@
 
 - `CartoBQTilerLayer` is removed. Use `CartoLayer` instead with `type` set to `MAP_TYPES.TILESET`.
```
Comment (ibgreen) [inline docs/upgrade-guide.md:7]: Specifically move these to the bottom of the upgrade guide

File: docs/whats-new.md (hunk 2 above)
Comment (felixpalmer) [reply docs/whats-new.md:11]: I fixed this in a separate PR https://github.com/visgl/deck.gl/pull/6680

File: docs/upgrade-guide.md (hunk 4 above)
Comment (felixpalmer) [reply docs/upgrade-guide.md:7]: Moved to bottom and placed in separate section

**Review by borja-munoz — APPROVED** [review]

**Review by ibgreen — APPROVED** [review]

**Review by Pessimistress — APPROVED** [review]

**Review by alasarr — CHANGES_REQUESTED** [review]

File: docs/whats-new.md [hunk 5]
```diff
@@ -2,6 +2,39 @@
...
+
+Release date: February 24, 2022
+
+### MaskExtension
+
+The new [MaskExtension](/docs/api-reference/extensions/mask-extension) allows layers to show/hide objects by a geofence. For example, a map may filter a list of user locations by the boundaries of a given country, or highlight part of a base map that is inside a user-drawn circle or lasso area.
+
+![Masking flights and cities in Texas](https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/googlemaps-scenegraph.png?raw=true)
+
+### @deck.gl/carto fetchMap
```
[the remark this answers is not in the document]
Comment (felixpalmer) [reply docs/whats-new.md]: Done. @borja-munoz could you just add the image?

**Review by alasarr — APPROVED** [review]

**Pull request merged**